core/src/main/java/com/linecorp/armeria/internal/PathAndQuery.java

```diff
@@ -365,7 +351,7 @@ private static boolean appendOneByte(Bytes buf, int cp, boolean wasSlash, boolea
             if (isPath) {
                 return false;
             } else if (cp != 0x0A && cp != 0x0D && cp != 0x09) {
-                // .. except 0x0A (LF), 0x0D (CR) and 0x09 (TAB) because they are used in a form.
+                // .. except 0x0A (LF), (byte) 0x0D (CR) and 0x09 (TAB) because they are used in a form.
```

Comment:
Is that a reason that you put `(byte)` only in front of `0x0D`?

Reply:
Replacement failure. 😅